resources/EntitiesAfterbirth.xml

```diff
@@ -211,11 +208,10 @@
 
 	<group Name="Scarred Womb Enemies">
 		<entity BaseHP="35" Tags="Champion" ID="58" Image="Entities/58.1.0 - Scarred Para-Bite.png" Name="Scarred Para-Bite" Subtype="0" Variant="1" />
-		<group Name="Scarred Guts">
+		<group Name="Scarred Guts" ShowInUI="1">
```

Comment:
I was thinking ShowInUI should translate to showing in the palette, so the inverse means the group is just semantic and can be used in the stats dialog. Do we have more than 1 layer of group nesting anywhere?

Reply:
Oops. Yes, we have a lot more than 1 layer of group nesting. Particularly with the floors tab, which uses groups to split up entities based on chapter > floor > type of entity.